pages/index.tsx

```diff
@@ -1,154 +1,128 @@
-              </FormControl>
-              <FormControl className={styles.filter} variant="outlined">
-                <InputLabel>Keyword</InputLabel>
-                <Select label="Keyword">
-                  <MenuItem value="">
-                    <em>None</em>
-                  </MenuItem>
-                  <MenuItem value={1}>One</MenuItem>
-                  <MenuItem value={2}>Two</MenuItem>
-                  <MenuItem value={3}>Three</MenuItem>
-                </Select>
-              </FormControl>
-              <FormControl className={styles.filter} variant="outlined">
-                <InputLabel>More</InputLabel>
-                <Select label="More">
-                  <MenuItem value="">
-                    <em>None</em>
-                  </MenuItem>
-                  <MenuItem value={1}>One</MenuItem>
-                  <MenuItem value={2}>Two</MenuItem>
-                  <MenuItem value={3}>Three</MenuItem>
-                </Select>
-              </FormControl>
-            </div>
-
-            <div className={styles.cards}>
-              {orgs.length !== 0 ? (
-                orgs.map((org) => (
-                  <Card className={styles.card} key={org.id}>
-                    <CardActionArea
-                      onClick={() => router.push(`/orgs/${org.id}`)}
-                    >
-                      <CardContent>
-                        <Typography variant="h5">{org.name}</Typography>
-                        <Typography variant="body2">
-                          {org.organizationType}
-                          {org.organizationType && org.workType ? ' • ' : null}
-                          {org.workType}
-                        </Typography>
-                      </CardContent>
-                    </CardActionArea>
-                  </Card>
-                ))
-              ) : (
-                <Typography>No Organizations</Typography>
-              )}
-            </div>
-          </div>
-
+      <form onSubmit={formik.handleSubmit}>
+        <div className={styles.root}>
+          <div className={styles.leftCol}>{slogan}</div>
           <div className={styles.rightCol}>
-            <Map orgs={orgs} width="100%" height="100%" />
+            <Card className={styles.searchCard}>
+              <div className={styles.big}>Explore Organizations</div>
+              <div className={styles.auto}>
+                <Autocomplete
+                  multiple
+                  id="tags-outlined"
+                  options={ages}
+                  getOptionLabel={(option) => option}
+                  filterSelectedOptions
+                  onChange={(event, newValue) => {
+                    formik.setFieldValue('ages', newValue);
+                  }}
+                  className={styles.autoField}
+                  renderInput={(params) => (
+                    <TextField
+                      {...params}
+                      variant="outlined"
+                      placeholder="Ages"
```

Comment:
am not 100% sure if this is the right spot, but (minor issue) placeholder copy should match figma ("by orientations/background/audience/name")